hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-globalpolicygenerator/src/test/java/org/apache/hadoop/yarn/server/globalpolicygenerator/applicationcleaner/TestDefaultApplicationCleaner.java

```diff
@@ -96,6 +119,13 @@ public void breakDown() {
       stateStore.close();
       stateStore = null;
     }
+    if (registryClient != null) {
+      registryClient.cleanAllApplications();
```

Comment:
Can we null the registryClient too?

Reply:
Thank you for your help in reviewing the code! I will improve this part of the code.